src/main/resources/db/migration/V0_1_0__create_user_table.sql

```diff
@@ -0,0 +1,5 @@
+Create TABLE `user` (
+    user_id VARCHAR(36) PRIMARY KEY,
```

Comment:
I wouldn't prefix columns names with tables name, use `id` and not `user_id`.
When choosing the name of a column try think of how it would be used, like: `user.id` and `user.user_id`, which is better? Now try `tweet.user_id` and `tweet.author_id`, which one makes more sense?